keps/sig-auth/20190916-noderestriction-pods.md

```diff
@@ -0,0 +1,476 @@
+---
+title: Extended NodeRestrictions for Pods
+authors:
+  - "tallclair"
+owning-sig: sig-auth
+participating-sigs:
+  - sig-node
+  - sig-cluster-lifecycle
+reviewers:
+  - TBD
+approvers:
+  - TBD
+editor: TBD
+creation-date: 2019-09-16
+status: implementable
```

Comment:
suggest marking provisional to unblock merge while gathering feedback from SCL and static pod authors

Reply:
It needs to be implementable by the 10/15 if it's going to make alpha in v1.17. Maybe it's already wishful thinking, but I don't see a second PR to mark implementable getting enough attention by then?